pubkey_encrypt.install

```diff
@@ -63,4 +63,20 @@ function pubkey_encrypt_install() {
 
   // Initialize all Role keys.
   $pubkey_encrypt_manager->initializeRoleKeys();
+  // Initialize corresponding Encryption Profiles for the Role keys.
+  $pubkey_encrypt_manager->initializeEncryptionProfiles();
+}
+
+/**
+ * Implements hook_uninstall().
+ */
+function pubkey_encrypt_uninstall() {
+  // Delete Encryption Profiles for all Role keys.
+  $keys = \Drupal::service('key.repository')
+    ->getKeysByProvider('pubkey_encrypt');
```

Comment:
Can probably keep this to one line if it's <= 80 characters and there's only one method call (no chaining).